SpecializedServers/Horizon/SERVER/Medius/MAPS.cs

```diff
@@ -95,6 +100,7 @@ protected override async Task ProcessMessage(BaseScertMessage message, IChannel
 
                             data.ClientObject = new(scertClient.MediusVersion ?? 0)
                             {
+                                MediusVersion = scertClient.MediusVersion ?? 0,
```

Comment:
This isn't necessary, we pass the argument in the constructor.

Reply:
Ahh yeah I see, forgot about that, I don't normally pass arguements in the constructor :)